src/answers-search-bar.js

```diff
@@ -247,11 +248,12 @@ class AnswersSearchBar {
     }
     parsedConfig.sessionTrackingEnabled = sessionTrackingEnabled;
 
+    const authIdKey = parsedConfig.apiKey ? 'apiKey' : 'token';
```

Comment:
I think we should make this behavior clear in the README. It's not an invalid state if you provide both an `apiKey` and a 'token`, we'll just ignore the `token`.

Reply:
Spec noted it would be an invalid state if both is provided. If that changed then I will update this